app/assets/stylesheets/components/_btn.scss

```diff
@@ -10,21 +10,11 @@
   // of a link.
   display: inline;
   width: auto;
-
-  // Temporary: To be backported to design system. Unstyled buttons should not ever show the default
-  // opaque disabled background color. Typically a button would not be both disabled and active, but
-  // it is possible in some browsers when e.g. disabling a button in response to its own click.
-  &.usa-button:disabled:hover,
-  &.usa-button:disabled.usa-button--hover,
-  &.usa-button:disabled:active,
-  &.usa-button:disabled.usa-button--active,
-  &.usa-button:disabled:focus,
-  &.usa-button:disabled.usa-focus {
-    background-color: transparent;
-  }
```

Comment:
This was patched upstream in https://github.com/uswds/uswds/pull/4077 , available as of USWDS 2.11.2 / [`identity-style-guide@6.0.0`](https://github.com/18F/identity-style-guide/releases/tag/v6.0.0).